authority-portal-backend/authority-portal-db/src/main/resources/db/testdata/V1_1__Initial_Schema_Test_Data.sql

```diff
@@ -24,8 +24,6 @@ values ('00000000-0000-0000-0000-000000000007', null, 'APPROVED');
 insert into "user" (id, organization_mds_id, registration_status)
 values ('00000000-0000-0000-0000-000000000008', null, 'PENDING');
 insert into "user" (id, organization_mds_id, registration_status)
-values ('00000000-0000-0000-0000-000000000009', null, 'PENDING');
-insert into "user" (id, organization_mds_id, registration_status)
```

Comment:
What does this change do?

Reply:
Funny you should ask... I think this was about a user in the DB that has no mdsId, which is an invalid state as of now.